src/test/java/com/opipo/codechallenge/config/TransactionToTransactionEntityMapIT.java

```diff
@@ -0,0 +1,81 @@
+package com.opipo.codechallenge.config;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.opipo.codechallenge.CodechallengeApplication;
+import com.opipo.codechallenge.repository.model.TransactionEntity;
+import com.opipo.web.api.model.Transaction;
+
+@SpringBootTest(classes = CodechallengeApplication.class)
+@DisplayName("TransactionToTransactionEntityMap Integration test")
+public class TransactionToTransactionEntityMapIT {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Test
+    @DisplayName("Given Transaction then transform it to TransactionEntity")
+    void givenTransactionThenTransformToTransactionEntity() {
+        String accountIban = "myaccount";
+        BigDecimal amount = BigDecimal.valueOf(48d);
+        OffsetDateTime date = OffsetDateTime.now();
+        String description = "my description";
+        BigDecimal fee = BigDecimal.valueOf(3.5d);
+        String reference = "referencee";
+        Transaction transaction = buildTransaction(accountIban, amount, date, description, fee, reference);
+
+        TransactionEntity actual = modelMapper.map(transaction, TransactionEntity.class);
+
+        checkTransactionEntityIsCorrect(actual, accountIban, amount, date, description, fee, reference);
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/albertinizao/codechallenge/pullRequest?prid=4681930)